iso15118/evcc/comm_session_handler.py

```diff
@@ -550,7 +550,10 @@ async def get_from_rcv_queue(self, queue: asyncio.Queue):
                 elif isinstance(notification, StopNotification):
                     await cancel_task(self.comm_session[1])
                     del self.comm_session
-                    if not notification.successful:
+                    if notification.successful:
+                        if not self.config.is_alive_after_session:
+                            break
```

Comment:
Following my suggestion from above, I think this would suffice. What do you think? 
```suggestion
                    if notification.successful:
                        break
```

Reply:
It looks good to me. Added another commit to fix all together.